airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -71,19 +62,20 @@ def get_jobs_for_job_group(self, job_id: int) -> Dict[str, Any]:
         :type job_id: int
         """
 
-        url: str = f"{self._url}/{job_id}/jobs"
+        endpoint_path = f"/v4/jobGroups/{job_id}/jobs"
+        url: str = f"{self._base_url}{endpoint_path}"
```

Comment:
```suggestion
        endpoint_path = f"v4/jobGroups/{job_id}/jobs"
        url: str = os.path.join(self._base_url, endpoint_path)
```
In this way we will handle `_base_url` with and without ending `/`

Reply:
Nice. 
I like it.